rebasehelper/helpers/lookaside_cache_helper.py

```diff
@@ -179,6 +179,9 @@ def post(check_only=False):
         logger.info('Uploading %s to lookaside cache', path)
         try:
             post()
+        except ConnectionError as e:
```

Comment:
Did you test this? Shouldn't it be `requests.exceptions.ConnectionError`?

Reply:
Good catch, didn't realize python has its own `ConnectionError`. I didn't manage to reproduce the issue but your remark makes sense